ProjectGroupA/src/main/java/com/fundation/search/Main.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
```

Comment:
For the whole headers, we will need to keep a consistency.
Paolo sent:
/**
 * @(#)AssetFile.java Copyright (c) 2018 Jalasoft.

Jose is using:
/*
 *  @(#)AssetFile.java Copyright (c) 2018 Jalasoft.